.github/pull_request_template.md

```diff
@@ -0,0 +1,17 @@
+<!--- TITLE -->
+
+#### Module/Algorithm
+- [ ] Core algorithm/module implementation
+- [ ] [Query module](https://memgraph.com/docs/memgraph/reference-guide/query-modules) implementation
+- [ ] Unit tests
+- [ ] End-to-end tests
+- [ ] Code documentation
+- [ ] README short description
+- [ ] Documentation on [memgraph/docs](https://github.com/memgraph/docs)
```

Comment:
Maybe the following parts are not obligatory:
unit tests, end-to-end, code documentation, and documentation on docs.

From discussion we had in August with Mislav I think we just want for now others to contribute.

Reply:
We want to sync with the contributors, and documentation would definitely help us understand their PRs quicker.
I’m not sure if requiring documentation would demotivate potential contributors as it’s a good practice to document your code anyway.

Agree with the rest of your comment! 👍

Reply:
Maybe we can include these as optional. My reasoning for including this stuff was to never again forget them once I merge the module. I've done it several times now. Do you have an idea of how should we have both clean structures without big requirements and with reminders of important stuff (tests, documentation).

Reply:
Would it make sense to have two templates?
Let’s give the contributors a basic template called something along the lines of “First PR”, and we’ll continue to use the “big” one.

Reply:
It's common to use only one file, feel free to organize it and propose a new solution for two templates within it

Reply:
I think this now is fine how you structured it. We want others to contribute, when they make PR we can ask them to add tests or we can add them. So I think I will approve this now.